Backend/app/rider-platform/rider-app/Main/spec/Storage/PartnerOrganization.yaml

```diff
@@ -0,0 +1,73 @@
+imports:
+  DbHash: Kernel.External.Encryption
+  Merchant: Domain.Types.Merchant
+  Value: Data.Aeson
+  APIRateLimitOptions: Kernel.Types.SlidingWindowLimiter
+  SmsSessionConfig: Kernel.Sms.Config
+
+PartnerOrganization:
+  tableName: partner_organization
+  derives: "Generic, Show, ToJSON, FromJSON"
+  fields:
+    orgId: Id PartnerOrganization
+    name: Text
+    apiKeyHash: DbHash
+    merchantId: Id Merchant
+    createdAt: UTCTime
+    updatedAt: UTCTime
+
+  types:
+    PartnerOrgTransaction: {}
+
+  constraints:
+    apiKeyHash: PrimaryKey
+    orgId: SecondaryKey
+
+  queries:
+    findByApiKeyHash:
+      kvFunction: findOneWithKV
+      where: apiKeyHash
+
+  sqlType:
+    apiKeyHash: text
+
+  excludedFields: [merchantOperatingCityId]
+
+PartnerOrgConfig:
+  tableName: partner_org_config
+  derives: "Generic, Show, ToJSON, FromJSON"
+  fields:
+    partnerOrgId: Id PartnerOrganization
+    config: PartnerOrganizationConfig
```

Comment:
This table also requires a key to lookup the specific config, instead of loading everything?